jdk/src/share/classes/java/util/concurrent/LinkedTransferQueue.java

```diff
@@ -698,7 +697,6 @@ else if (!p.casNext(null, s))
      * @return matched item, or e if unmatched on interrupt or timeout
```

Comment:
@chrisr3 - perhaps we should update the `@param timed` to indicate that the param is ignored?